src/subclass/cell_renderer.rs

```diff
@@ -0,0 +1,658 @@
+use cairo_sys;
+use gdk_sys;
+use glib_sys;
+use gtk_sys;
+
+use libc::{c_char, c_int};
+use std::mem;
+
+use cairo;
+use glib::object::IsA;
+use glib::subclass::prelude::*;
+use glib::translate::*;
+use glib::ObjectClass;
+use glib::GString;
```

Comment:
Misses a run of `cargo fmt`

Reply:
yeah I realized afterwards I forgot one last round.